src/table.tsx

```diff
@@ -19,21 +19,25 @@ function Table({ data, numberFormatter = String }: TableProps): VNode {
       <table className="usa-table usa-table--compact">
         <thead>
           <tr>
-            {header.map((head) => (
-              <th>{head}</th>
-            ))}
+            {header.map((head) =>
+              typeof head === "object" && head.type === "th" ? head : <th>{head}</th>
```

Comment:
Another Preact difference: In React we could use [`isValidElement`](https://reactjs.org/docs/react-api.html#isvalidelement) 😅

Reply:
is it time to switch? i don't feel strongly about either react or preact, I just like the sound of "smaller"

Reply:
> is it time to switch? i don't feel strongly about either react or preact, I just like the sound of "smaller"

This one's a pretty minor thing, I'm not too concerned. It's a pretty rare use-case and in a sense I have some appreciation for Preact's leanness. Personally I'd be fine to stay on Preact, and I think it could be a helpful stress test if we'd want to leverage it in other settings like [design system](https://github.com/uswds/uswds/issues/3445#issuecomment-674060672) [custom elements](https://preactjs.com/guide/v10/web-components/). The point of being "smaller" has some irony though, because at the same time we pull in [_enormous_](https://bundlephobia.com/package/@observablehq/plot) D3 libraries 😄